src/functional/ffront/source_utils.py

```diff
@@ -21,16 +21,19 @@
 import textwrap
 from collections.abc import Callable, Iterator
 from dataclasses import dataclass
-from typing import Any, Union, cast
+from typing import Any, cast
 
-from eve import extended_typing as xtyping
 from functional import common
-from functional.ffront import fbuiltins
 
 
 MISSING_FILENAME = "<string>"
 
 
+def get_external_vars(function: Callable) -> dict[str, Any]:
```

Comment:
```suggestion
def get_external_vars_from_function(function: Callable) -> dict[str, Any]:
```
The name is too unspecific to me (given that there is (usually) no dispatch on the arguments type in Python). In any case the convention below is different.